### PR TITLE
none: adding some basic mail related config fs ft

### PR DESCRIPTION
**Pull request by Shane-XB-Qian**
- notmuch
- mbsync
- msmtp

### CODE REVIEW

**Comment by ericpruitt** [comment]
There's already an msmtprc syntax file distributed with the code: <https://fossies.org/linux/msmtp/scripts/vim/msmtp.vim>

**Comment by Shane-XB-Qian** [comment]
@ericpruitt oh, thank you, i do not know, and seems it did not include in vim code base, and looks you were the author of this syntax file, would you mind i include it within this PR, or you would do it by youself, or neither?
// for now this PR is just for basic syntax, if you have professional one, then would be better.

**Comment by Shane-XB-Qian** [comment]
@ericpruitt perhaps this time let it just be for basic one,
// vs you can change it to your version when you have time and if ok to include it to vim code base,
// and i supposed you have others runtime files for mstmp too (other than syntax).
so this PR should be ready.

**Comment by chrisbra** [comment]
thanks!

**Comment by chrisbra** [comment]
Thanks, I'll make it then 'mstmp' for '.msmtprc', without including a syntax file. @ericpruitt feel free to submit your that syntax file for distribution with Vim. Thanks!

File: runtime/filetype.vim [hunk 1]
```diff
@@ -1410,6 +1414,10 @@ au BufNewFile,BufRead *.mix,*.mixal		setf mix
 " MMIX or VMS makefile
 au BufNewFile,BufRead *.mms			call dist#ft#FTmms()
 
+" msmtp
+" TODO: just for generic syntax for now
+au BufNewFile,BufRead .msmtprc			setf conf
```
Comment (Konfekt) [inline runtime/filetype.vim:1419]: This now  moved to ~/.config/msmtp/config. See also https://github.com/marlam/msmtp/pull/148/files

**Comment by Shane-XB-Qian** [comment]
> This now  moved to ~/.config/msmtp/config. See also https://github.com/marlam/msmtp/pull/148/files

it actually was a symblic ~/.msmtprc, or anyway:

```patch
diff --git a/runtime/filetype.vim b/runtime/filetype.vim
index 927230d4f..7da453365 100644
--- a/runtime/filetype.vim
+++ b/runtime/filetype.vim
@@ -1433,12 +1433,13 @@ au BufNewFile,BufRead *.mix,*.mixal		setf mix

 " MMIX or VMS makefile
 au BufNewFile,BufRead *.mms			call dist#ft#FTmms()

 " msmtp
 au BufNewFile,BufRead .msmtprc			setf msmtp
+au BufNewFile,BufRead */.config/msmtp/config	setf msmtp

 " Symbian meta-makefile definition (MMP)
 au BufNewFile,BufRead *.mmp			setf mmp

 " ABB Rapid, Modula-2, Modsim III or LambdaProlog
 au BufNewFile,BufRead *.mod\c			call dist#ft#FTmod()
```

-- 
shane.xb.qian

File: runtime/filetype.vim (hunk 1 above)
Comment (Shane-XB-Qian) [reply runtime/filetype.vim:1419]: it actually was a symblic  ~/.msmtprc

**Pull request closed without merging**